test/sqllogictest/dates-times.slt

```diff
@@ -725,11 +725,21 @@ SELECT current_timestamp > TIMESTAMP '2016-06-13 00:00:00'
 ----
 true
 
-query error calls to mz_logical_timestamp in in static or write queries are not supported
-CREATE VIEW timeview AS SELECT mz_logical_timestamp()
+statement ok
+CREATE VIEW logical_timestamp_view AS SELECT mz_logical_timestamp()
 
-query error now cannot be used in static queries
-CREATE VIEW timeview AS SELECT now()
+query T
+SELECT * FROM logical_timestamp_view
+----
+18446744073709551615
```

Comment:
I don't understand how this could be a fixed value; would you mind leaving a comment for me and future mortals we hire?

Reply:
It's exactly equivalent to executing `SELECT mz_logical_timestamp()`, which has the same somewhat confusing output. The idea is that because your query depends on no sources, its value is fixed for all time, and so we report the maximum possible time as the time at which the query executed.

Reply:
(I added a comment that describes that this is exactly equivalent to `SELECT mz_logical_timestamp()`; didn't want to explain the semantics of `mz_logical_timestamp` here though.)

Reply:
tysm